2-Authorization/3-call-own-api-dotnet-core-daemon/README.md

```diff
@@ -171,6 +171,8 @@ Open the project in your IDE (like Visual Studio or Visual Studio Code) to confi
 1. Find the placeholder `Enter_the_Tenant_Id_Here` and replace the existing value with your Azure AD tenant/directory ID.
 1. Find the placeholder `Enter_the_Tenant_Subdomain_Here` and replace it with the Directory (tenant) subdomain. For instance, if your tenant primary domain is *contoso.onmicrosoft.com*, use *contoso*.
 
+    To use a CIAM custom domain instead, you can replace the entire `https://Enter_the_Tenant_Subdomain_Here.ciamlogin.com/` placeholder with the (custom domain)/tenant/v2.0. For instance, if you custom domain is *contoso.ciamcustom.com*, you can use `https://contoso.consotoextensibility.com/tenant/v2.0`.
```

Comment:
Maybe add this as an important note

> [!IMPORTANT]  
> If you are using a Custom Domain ...

Reply:
> [!IMPORTANT]  
doesn't actually work in GH doc files

Reply:
I think making it a quote make sit look a bit more obvious. But yeah, i can add a not somewhere else too.
![image](https://github.com/Azure-Samples/ms-identity-ciam-dotnet-tutorial/assets/30090357/ace9d93d-d1c7-48eb-a63a-674c8f04ca61)